.github/workflows/collector-full.yml

```diff
@@ -221,50 +237,3 @@ jobs:
           dst-image: quay.io/rhacs-eng/collector:${{ inputs.collector-tag }}-latest
           username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
           password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
-
-  retag-collector-slim:
```

Comment:
I long for a day where retagging is unneeded (in a post-ebpf world?) because I find it painfully difficult to visualise. Moving this to ansible is a great step though, and simplifies things nicely.

Reply:
You and me both buddy, we have way too many edge cases attached to the full collector image. For instance, I just realized master fails to build s390x and ppc64le on master when we bump the driver version because the drivers have not been built downstream and pushed back up for us to use.